sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2918,6 +2918,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val PYTHON_TABLE_UDF_ARROW_ENABLED =
+    buildConf("spark.sql.execution.pythonUDTF.arrow.enabled")
+      .doc("Enable Arrow optimization for Python UDTFs.")
+      .version("3.5.0")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
@HyukjinKwon Arrow optimization is enabled by default for Python UDTFs. If the user does not have the necessary pyarrow dependency, it will fall back to using regular Python UDTFs. Do you think this is viable?

Reply:
I don't think we can do it. We should know if there's Arrow installed in all executors. We could optionally fallback in executors but I think there'd be a lot of corner cases to deal with, and I don't think it's worthwhile.